### DISCUSSION TITLE
revChatGPT.V1.Error: OpenAI: {"detail":{"message":"This key is associated with a deactivated account. If you feel this is an error, contact us through our help center at help.openai.com.","type":"invalid_request_error","param":null,"code":"account_deactivated"}} (code: 401)

### DISCUSSION THREAD

**Discussion by xxgzzp** [acheong08/ChatGPT#1156] [General]
revChatGPT.V1.Error: OpenAI: {"detail":{"message":"This key is associated with a deactivated account. If you feel this is an error, contact us through our help center at help.openai.com.","type":"invalid_request_error","param":null,"code":"account_deactivated"}} (code: 401)

How to unseal

**Comment by hrizal** [comment]
me too
